content/en/docs/tasks/observability/metrics/telemetry-api/index.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Customizing Istio Metrics with Telemetry API
+description: This task shows you how to customize the Istio metrics with Telemetry API.
+weight: 10
+keywords: [telemetry,metrics,customize]
+owner: istio/wg-policies-and-telemetry-maintainers
+test: no
+---
+
+Telemetry API has been in Istio as a first-class API for quite sometime now.
+Previously users had to configure metrics in the `telemetry` section of Istio configuration,
+This task shows you how to customize the metrics that Istio generates with Telemetry API.
+
+## Before you begin
+
+[Install Istio](/docs/setup/) in your cluster and deploy an application.
+Telemetry API can not work together with the `EnvoyFilter` way.
+From `1.18`, stats `EnvoyFilter` will not be installed by default.
+For version before `1.18`, you should install with following configuration in `IstioOperator` configuration:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    defaultConfig:
+      extraStatTags:
+        - source_x
+        - destination_x
+  values:
+    telemetry:
+      enabled: true
+      v2:
+        enabled: false
```

Comment:
Thanks for the PR!  To help readers understand the configuration here, in the paragraph above, can you add an explanation of why telemetry.v2.enabled is set as false?